src/IRPrinter.cpp

```diff
@@ -425,21 +428,31 @@ void IRPrinter::visit(const Call *op) {
     }
 
     stream << op->name << "(";
-    for (size_t i = 0; i < op->args.size(); i++) {
-        print(op->args[i]);
-        if (i < op->args.size() - 1) {
-            stream << ", ";
-        }
-    }
+    indent += 2;
+     for (size_t i = 0; i < op->args.size(); i++) {
+        stream << "\n";
+        do_indent();
+         print(op->args[i]);
+         if (i < op->args.size() - 1) {
+             stream << ", ";
+         }
+     }
+    indent -= 2;
+    stream << "\n";
+    do_indent();
     stream << ")";
 }
 
 void IRPrinter::visit(const Let *op) {
     stream << "(let " << op->name << " = ";
     print(op->value);
-    stream << " in ";
+    stream << " in";
```

Comment:
Yeah, I hear you. This formatting does help me, but I didn't intent do include it into this PR. Reverted.